src/zenml/models/v2/base/scoped.py

```diff
@@ -486,16 +490,41 @@ class TaggableFilter(BaseFilter):
     tag: Optional[str] = Field(
         description="Tag to apply to the filter query.", default=None
     )
+    tags: Optional[List[str]] = Field(
```

Comment:
`tag` and `tags`? :D

Reply:
I initially wanted to implement `tags` and deprecate `tag`. However, later on, I've realized that we are currently not dealing with the CLI case where the filter models would have multiple entries for the same attribute. Until that is properly fixed, I decided to remove the deprecation of the `tag` attribute here. WDYT?

Reply:
Shouldn't we fix that as part of this PR then?

Reply:
I fixed this as well. Now, `tag` option is deprecated for all `TaggableFilter`s and you can use multiple entries for all the attributes in the Filter models with `zenml artifact version list --tags="startswith:a" --tags="s"`. WDYT?